### ISSUE TITLE
Use of new Uberon terms

### ISSUE THREAD

**Issue by fbastian** [BgeeDB/expression-annotations#112]
@ANiknejad: you requested new terms to Uberon for some annotations, were all our annotations updated after these new terms were added to Uberon?

**Comment by ANiknejad** [comment]
yes but they were checked on master/annotations/rna_seq/RNASeqLibrary.tsv

Here the list of new terms already annotated in master/annotations/rna_seq/RNASeqLibrary.tsv

UBERON:0002098	apex of heart
UBERON:0035841	esophagogastric junction muscularis propria
UBERON:0035805	muscle layer of sigmoid colon

The following terms are not used in RNASeqLibrary.tsv currently but should, see new issue #113

UBERON:0035825      left adrenal gland cortex
for example should be used to annotate sample
GTEX-XUJ4-0126-SM-4BOP7

UBERON:0035827   right adrenal gland cortex

**Comment by ANiknejad** [comment]
done in issue #113

**Comment by ANiknejad** [comment]
id: UBERON:0035835
name: apical region of left ventricle

Chris also created this new term for us, I  used it to re-annotate GTEx sample:
GTEX-OIZG-1126-SM-2HMIU
(the only one concerned by this annotation)

**Comment by ANiknejad** [comment]
@julien-roux 
as told you just now
